src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1793,6 +1816,9 @@ void goto_instrument_parse_optionst::help()
     " --show-local-safe-pointers   show pointer expressions that are trivially dominated by a not-null check\n" // NOLINT(*)
     " --show-safe-dereferences     show pointer expressions that are trivially dominated by a not-null check\n" // NOLINT(*)
     "                              *and* used as a dereference operand\n" // NOLINT(*)
+    " --show-index-exprs array_name\n"
```

Comment:
Again, this appears unrelated.

Reply:
This too was for our internal tests. Removed it from the PR.

Reply:
Removed this option from PR.